openssl3/fulltest/testrun.py

```diff
@@ -10,22 +10,28 @@
    jsoncontents = f.read();
 
 assignments = json.loads(jsoncontents)
+ok=True
 for sig in assignments:
     print("Testing %s:" % (sig))
     for kem in assignments[sig]:
-     if not "_" in kem: # hybrids not yet supported in OSSL3-oqsprovider: TBD
+    # if not "_" in kem: # hybrids not yet supported in OSSL3-oqsprovider: TBD
```

Comment:
Will this code actually run after line 17 is commented out? Indentation is different once this is commented out, not sure if that's still valid Python.

Reply:
I'd say so -- as long as nobody moves the "if" statement to the same level as the "for".

Reply:
More specifically will the code underneath it still run now that it's at the wrong indentation level?  If you executed the script already then my question can be ignored.

Reply:
Yes, absolutely. You can give it a try with `./testrun.sh openquantumsafe/oqs-ossl3`

Reply:
But actually, I didn't really mean to include this change in the PR :-( All I wanted to change is the ossl3 Dockerfile. So this is an error. PR will be updated, eliminating this.